src/annoymodule.cc

```diff
@@ -250,6 +250,19 @@ py_an_build(py_annoy *self, PyObject *args) {
 
 
 static PyObject *
+py_an_unbuild(py_annoy *self, PyObject *args) {
+  if (!self->ptr) 
+    Py_RETURN_NONE;
+  
+  Py_BEGIN_ALLOW_THREADS;
```

Comment:
cool yeah i don't think you need to go crazy with it, just run `build` and `unbuild` a bunch of times and make sure things don't crash :)